ksqldb-rest-client/src/main/java/io/confluent/ksql/rest/client/KsqlTarget.java

```diff
@@ -221,6 +222,21 @@ public RestResponse<List<StreamedRow>> postQueryRequest(
         KsqlTarget::toRows);
   }
 
+  public RestResponse<List<StreamedRow>> postQueryStreamRequestProto(
+          final String ksql,
+          final Map<String, ?> requestProperties
+  ) {
+    final Map<String, Object> requestPropertiesObject = requestProperties.entrySet().stream()
```

Comment:
Can we make `requestProperties` of type `Map<String, Object>`  instead of creating a full copy?

Reply:
done